File: ignite-nest/src/comment/entity/comment.entity.ts
import { Column, Entity, ManyToOne } from 'typeorm';
import { Base } from '../../common/entity/base';
import { Post } from '../../post/entity/post.entity';

@Entity('comment')
export class Comment extends Base {
  @Column({ type: 'varchar', length: 50, name: 'title' })
  title: string;

  @Column({ type: 'text', name: 'content' })
  content: string;

  @ManyToOne(() => Post, (post) => post.comments)
  post: Post;
}

File: ignite-spring/src/main/kotlin/org/rookedsysc/ignitespring/comment/entity/CommentEntity.kt
package org.rookedsysc.ignitespring.comment.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import org.rookedsysc.ignitespring.common.entity.BaseEntity

@Entity(name = "comment")
class CommentEntity(
    @Column(name = "post_id", nullable = false)
    var postId: Long,

    @Column(columnDefinition = "varchar(50)", length = 50, nullable = false)
    var title: String,

    @Column(columnDefinition = "text", nullable = false)
    var content: String,
    ) : BaseEntity(
) {}
